suite-native/module-accounts/src/components/AccountsList.tsx

```diff
@@ -4,12 +4,23 @@ import { useSelector } from 'react-redux';
 import { A } from '@mobily/ts-belt';
 
 import { Box, Text } from '@suite-native/atoms';
-import { selectAccounts } from '@suite-common/wallet-core';
+import {
+    AccountsRootState,
+    selectAccounts,
+    selectAccountsByNetworkSymbols,
+} from '@suite-common/wallet-core';
+import { NetworkSymbol } from '@suite-common/wallet-config';
 
 import { AccountListItem } from './AccountListItem';
 
-export const AccountsList = () => {
-    const accounts = useSelector(selectAccounts);
+type AccountsListProps = {
+    assets: NetworkSymbol[];
+};
+
+export const AccountsList = ({ assets }: AccountsListProps) => {
+    const accounts = useSelector((state: AccountsRootState) =>
+        A.isEmpty(assets) ? selectAccounts(state) : selectAccountsByNetworkSymbols(state, assets),
```

Comment:
I not sure about this, but probably OK

Reply:
Probably better solution will be to have somewhere array of all enabled networks for mobile app and it will be passed to `selectAccountsByNetworkSymbols` which will select everything. We don't have to do it know but it would be definitely useful because we already manually defining these enabled `btc` and `test` in quite a lot places now.

Reply:
I added the comment there.